Packages/com.unity.inputsystem/InputSystem/Controls/Processors/EditorWindowSpaceProcessor.cs

```diff
@@ -19,13 +19,21 @@ namespace UnityEngine.InputSystem.Processors
     /// <seealso cref="Pointer.position"/>
     [DesignTimeVisible(false)]
     [Scripting.Preserve]
-    internal class EditorWindowSpaceProcessor : InputProcessor<Vector2>
+    public class EditorWindowSpaceProcessor : InputProcessor<Vector2>
     {
+        /// <summary>
+        /// Transform the given player screen-space coordinate into the coordinate space of the current
+        /// <c>EditorWindow</c>.
+        /// </summary>
+        /// <param name="value">GameView screen space coordinate.</param>
+        /// <param name="control">Ignored.</param>
+        /// <returns>The given coordinate transformed into <c>EditorWindow</c> space.</returns>
+        /// <remarks>
+        /// This method will only succeed if the editor is currently in an <c>EditorWindow</c> callback such
+        /// as <c>OnGUI</c>.
+        /// </remarks>
         public override Vector2 Process(Vector2 value, InputControl control)
         {
-            if (control == null)
```

Comment:
Do we not mind if it is null anymore?

Reply:
Yup, correct. It used to rely on the control to get to the device on which the position request was issued and then would send the coordinate mapping command to that device. That turned out to be nonsense as native backends then needed to implement the same command for every single type of pointer device they supported. So now we just send the command to the system mouse and ignore the control we get.